packages/eas-cli/src/build/metadata.ts

```diff
@@ -26,10 +26,7 @@ export async function collectMetadataAsync<T extends Platform>(
 ): Promise<Metadata> {
   const vcsClient = getVcsClient();
   const channelOrReleaseChannel = await resolveChannelOrReleaseChannelAsync(ctx);
-  const distribution =
-    ('simulator' in ctx.buildProfile && ctx.buildProfile.simulator
-      ? 'simulator'
-      : ctx.buildProfile.distribution) ?? 'store';
+  const distribution = ctx.buildProfile.distribution ?? 'store';
```

Comment:
This change will break stuff like `eas build:run` 🤔 

I'm wondering if altering the `distributionType` behavior is necessary 🤔.

 Maybe we can add some new field to Metadata to keep the raw `buildProfile.distribution` value there? I mean it's probably not optimal but I feel that this change can introduce some regressions in some parts of the codebase.
 
 What do you think?

Reply:
Can you elaborate on why `eas build:run` would be broken by this?
As for another field for "original distribution" or whatever we want to call it, I'm open for discussion but it feels to me like hacking around an existing hack (overwriting the distribution). I feel like we should remove the reliance on this hardcoded `simulator` value for `distribution` instead of building upon it, wdyt?

Reply:
I see, it runs a simulator build and fetches it by filtering by distribution = simulator.
Maybe it should handle both options (distribution = simulator or simulator = true) for a while instead of introducing another field?

Reply:
I concur, we should list builds with both `simulator: true` and `distribution: simulator`.

Reply:
It's handled in https://github.com/expo/universe/pull/13524